app/src/ui/add-repository/create-repository.tsx

```diff
@@ -271,6 +272,7 @@ export class CreateRepository extends React.Component<
 
     this.setState({ creating: true })
 
+    this.props.dispatcher.closeFoldout(FoldoutType.Repository)
```

Comment:
I think this line should run later in the process, after the git repo has been successfully created. I'd say right above the `this.props.dispatcher.selectRepository(repository)` from line 396

Reply:
Done